custom_components/adjustable_bed/beds/okimat.py

```
@@ -143,90 +234,136 @@ def _get_move_command(self) -> int:
+                command += remote.head_down
+
+        # Feet motor (93332 only)
+        if remote.feet_up is not None and remote.feet_down is not None:
+            if state.get("feet") is True:
+                command += remote.feet_up
+            elif state.get("feet") is False:
+                command += remote.feet_down
+
         return command
 
     async def _move_motor(self, motor: str, direction: bool | None) -> None:
```

Ruff (0.14.11)

Boolean-typed positional argument in function definition

(FBT001)